.github/workflows/samples-CalculatorCoreAppNuGet-ci-upgrade.yml

```diff
@@ -0,0 +1,65 @@
+name: CalculatorCoreAppNuGet CI (Upgrade)
+
+on:
+  schedule:
+    - cron: '0 15 * * 2' # Tuesday 3PM UTC (8AM PST)
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/samples-CalculatorCoreAppNuGet-ci-upgrade.yml'
+
+jobs:
+  build:
+    name: Build
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+      matrix:
+        sample: [CalculatorCoreAppNuGet]
+        architecture: [x86, x64, ARM64]
+        configuration: [Debug, Release]
+        reactNativeWindowsVersion: [latest, preview, canary]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Gather environment info
+        run: npx envinfo
+
+      - name: Relocate to reduce long paths
+        run:  Copy-item -Force -Recurse -Verbose "samples\${{ matrix.sample }}" -Destination "..\..\src"
+
+      - name: Yarn install
+        run: yarn install
+        working-directory: ..\..\src
+
+      - name: Upgrade react-native-windows
+        id: upgrade
+        continue-on-error: true
+        run: ${{github.workspace}}\.github\scripts\UpgradeSmokeTest.ps1 ${{ matrix.reactNativeWindowsVersion }} -Force ${{ format('${0}', contains(matrix.reactNativeWindowsVersion, 'canary')) }}
+        working-directory: ..\..\src
+
+      - name: Run react-native-windows-init

```

Comment:
I know you're already merged this, but you're gonna want to comment out the `react-native-windows-init` task here until you get the template created. Otherwise this is just going to fail.